devdocs/dependencies.md

```diff
@@ -0,0 +1,44 @@
+
+# Managing Server-Side Dependencies
+
+TEAMMATES uses Gradle to manage dependencies to external Java libraries.
+
+- [Adding new libraries](#adding-new-libraries)
+- [Updating libraries](#updating-libraries)
+
+## Adding new libraries
+
+There are two factors to consider:
+- Whether the library is for production or non-production code
+- Whether the library is needed for compile-time or runtime only
+
+Add the library in the appropriate section in the `build.gradle` file.
+
+## Updating libraries
+
+> Note the following:
+- Change `./gradlew` to `gradlew.bat` in Windows.
```

Comment:
I'm guessing older Windows version didn't support that. http://superuser.com/questions/319430/run-a-bat-file-without-typing-the-extension